### PR TITLE
docs: make mpy-cross

### PR DESCRIPTION
**Pull request by RandyMcMillan**

### CODE REVIEW

**Comment by RandyMcMillan** [comment]
This may belong somewhere else in the docs but It does need to happen prior to building the simulator.

**Comment by doc-hex** [comment]
I believe "make setup" does this already.

**Comment by achow101** [comment]
> I believe "make setup" does this already.

Doesn't seem like it.

**Comment by RandyMcMillan** [comment]
added command to 
```
make tools
```

File: README.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ Then:
 
 Setup and Run the Desktop-based Coldcard simulator:
 
+- `pushd external/micropython/mpy-cross && make && popd`
```
Comment (RandyMcMillan) [inline README.md:54]: Included in README.md for completeness

File: unix/Makefile [hunk 2]
```diff
@@ -29,6 +29,7 @@ tags:
 # Make a vanilla copy of micropython (unix) for use with upip and comparsion purposes
 tools:
 	cd $(PORT_TOP) && $(MAKE) clean deplibs all
+	pushd $(MPY_TOP)/mpy-cross && make && popd
```
Comment (RandyMcMillan) [inline unix/Makefile:32]: Included in make tools for completeness

**Comment by RandyMcMillan** [comment]
tested on macOS x86_64

with #78 

https://github.com/RandyMcMillan/firmware/commits/make-mp-cross-w-pr-78

File: unix/Makefile (hunk 2 above)
Comment (doc-hex) [reply unix/Makefile:32]: IIRC each command line of a makefile runs in a fresh shell, so no need to cd back. remove push/popd (like line above it)

File: README.md (hunk 1 above)
Comment (doc-hex) [reply README.md:54]: pretty sure "make setup" in ../unix does this already, but haven't checked

**Pull request closed without merging**